src/iso_alloc.c

```diff
@@ -645,6 +617,42 @@ INTERNAL_HIDDEN iso_alloc_zone_t *_iso_new_zone(size_t size, bool internal, int3
     return new_zone;
 }
 
+/* Fixes the next_sz_index list for a given zone */
```

Comment:
"Fixes" doesn't means much in this context: what does it fix?

Reply:
In this context 'fixes' means 'updates the list of zones for this size'. I can make the comment more accurate in the next PR.